dataland-frontend/src/components/resources/frameworkDataSearch/lksg/DataModelsTranslations.ts

```diff
@@ -1,378 +1,22 @@
-  forcedLabourPolicy: "Does your company have a Forced Labour Policy? If yes, please provide us with the policy.",
-  childLabourPolicy: "Does your company have a Child Labour Policy? If yes, please provide us with the policy.",
-  environmentalImpactPolicy:
-    "Does your company have a Environmental Impact Policy? If yes, please provide us with the policy.",
-  supplierCodeOfConduct:
-    "Does your company have a Supplier Code Of Conduct? If yes, please provide us with the Supplier Code Of Conduct.",
-  productionSiteName: "Production Site Name",
-  listOfProductionSites:
-    "Is your company a manufacturing company? If yes, please provide the names of the production sites.",
-  isInHouseProductionOrIsContractProcessing: "Is the production in-house or is the production done via subcontracting?",
-  addressesOfProductionSites:
-    "Addresses of the production sites (format: street, house number, zip code, city, country)",
-  listOfGoodsOrServices: "Which goods/services are manufactured or provided at the (production) sites?",
-  inHouseProductionOrContractProcessing: "Is in-house production or is Contract Processing",
-};
-
 export const listOfProductionSitesConvertedNames = {
-  name: "Name",
-  isInHouseProductionOrIsContractProcessing: "Is In-house Production Or Is Contract Processing",
-  country: "Country",
-  city: "City",
-  streetAndHouseNumber: "Street and house number",
-  postalCode: "Postal Code",
+  nameOfProductionSite: "Name",
+  addressOfProductionSite: "Address",
   listOfGoodsOrServices: "List Of Goods Or Services",
 };
 
-export const lksgSubAreasNameMappings = {
-  anticorruptionAndAntibribery: "Anti-corruption and anti-bribery",
-  biodiversity: "Biodiversity",
-  childLabour: "Child labour",
-  emissions: "Emissions",
-  energyPerformance: "Energy performance",
-  evidenceCertificatesAndAttestations: "Evidence, certificates and attestations",
-  forcedLabourSlaveryAndDebtBondage: "Forced labour, slavery and debt bondage",
-  freedomOfAssociation: "Freedom of association",
-  _general: "General",
-  greenhouseGasEmissions: "Greenhouse gas emissions",
-  greenSecurities: "Green securities",
-  grievanceMechanism: "Grievance mechanism",
-  humanRights: "Human rights",
-  osh: "OSH",
-  codeOfConduct: "Code Of Conduct",
-  riskManagement: "Risk management",
-  socialAndEmployeeMatters: "Social and employee matters",
-  water: "Water",
-  waste: "Waste",
-  environment: "Environment",
+export const detailsCompanyDataTableColumnHeaders = {
+  listOfProductionSites: {
+    nameOfProductionSite: "Name",
+    addressOfProductionSite: "Address",
+    listOfGoodsOrServices: "List Of Goods Or Services",
+  },
+  industry: {
+    industry: "NACE Code",
+  },
+  subcontractingCompaniesCountries: {
+    subcontractingCompaniesCountries: "Country",
+  },
+  subcontractingCompaniesIndustries: {
+    subcontractingCompaniesIndustries: "NACE Code",
+  },
```

Comment:
does
```
export const listOfProductionSitesConvertedNames = {
  nameOfProductionSite: "Name",
  addressOfProductionSite: "Address",
  listOfGoodsOrServices: "List Of Goods Or Services",
};

export const detailsCompanyDataTableColumnHeaders = {
  listOfProductionSites: {
    listOfProductionSitesConvertedNames
  },
  industry: {
    industry: "NACE Code",
  },
  subcontractingCompaniesCountries: {
    subcontractingCompaniesCountries: "Country",
  },
  subcontractingCompaniesIndustries: {
    subcontractingCompaniesIndustries: "NACE Code",
  },
};
```
work as well?

Reply:
No, but "listOfProductionSitesConvertedNames" is unnecessary and will be removed

Reply:
:-)

Reply:
ok